### PR TITLE
Wireframes

### PR DESCRIPTION
**Pull request by jonnyelliot**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
```
Comment (chrisflatley) [inline .idea/.gitignore:2]: Gitignore the .idea directory

File: frontend/frontend.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chrisflatley) [inline frontend/frontend.iml:1]: git ingore .iml files.

File: frontend/web/src/components/DayActivityList/DayActivityList.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import {
+  Box,
+  Button,
+  Divider,
+  Flex,
+  Heading,
+  IconButton,
+  Icons,
+} from '@committed/components'
+import moment from 'moment'
```
Comment (chrisflatley) [inline frontend/web/src/components/DayActivityList/DayActivityList.tsx:10]: Replace moment  as its so big (https://github.com/you-dont-need/You-Dont-Need-Momentjs) - which is also echoed on the momentjs site as well.

Looking at that table, dayjs seems an obvious one to go for  https://github.com/iamkun/dayjs (lots of stars, and smallest)

File: template.iml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chrisflatley) [inline template.iml:1]: Odd this is called template.iml?  I assume the pom.xml and package.json has been renamed [I've not checked]

File: frontend/web/package.json [hunk 5]
```diff
@@ -9,6 +9,7 @@
     "@committed/layout": "^2.0.1",
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
+    "moment": "^2.29.1",
```
Comment (chrisflatley) [inline frontend/web/package.json:12]: replace moment as per other comment.

**Review by chrisflatley — CHANGES_REQUESTED** [review]

File: .idea/.gitignore (hunk 1 above)
Comment (jonnyelliot) [reply .idea/.gitignore:2]: From the template ticket: 

> The current gitignore specifically ignores certain files in the intellij .idea folder, which seems a deliberate choice? Maybe we shouldnt ignore the entire .idea folder?

Comment (chrisflatley) [reply .idea/.gitignore:2]: ignore it - no good every comes from IDE files in the repo.

File: frontend/web/src/components/DayActivityList/DayActivityList.tsx (hunk 3 above)
Comment (jonnyelliot) [reply frontend/web/src/components/DayActivityList/DayActivityList.tsx:10]: I've created a ticket https://github.com/commitd/daze/issues/4

**Pull request merged**